doc/architecture/architecture_overview.adoc

```diff
@@ -0,0 +1,36 @@
+= Overview of OSI architecture
+
+OSI contains an object-based environment description using the message format of the https://github.com/protocolbuffers/protobuf/wiki[Protocol Buffer] library developed and maintained by Google.
+OSI defines top-level messages that are used to exchange data between separate models.
+Top-level messages define the ``GroundTruth`` interface, the ``SensorData`` interface and, since OSI version 3.0.0, the ``SensorView``, ``SensorViewConfiguration`` and ``FeatureData`` interfaces.
```

Comment:
TrafficCommand and TrafficUpdate missing.

Reply:
Remove comment, gets clear below.

Reply:
I also fell over that here: https://github.com/OpenSimulationInterface/open-simulation-interface/pull/535/files
let's add an 
Top-level messages **regarding sensor models** define...